### PR TITLE
fix mypy issue

### PR DESCRIPTION
**Pull request by cqc-melf**

### CODE REVIEW

**Review by cqc-alec — COMMENTED** [review]
Do we have `warn_unused_ignores = True` in this repo?

**Comment by cqc-melf** [comment]
> Do we have `warn_unused_ignores = True` in this repo?

I have added that now

File: mypy.ini [hunk 1]
```diff
@@ -19,6 +19,8 @@ warn_no_return = False
 warn_return_any = True
 warn_unreachable = True
 
+warn_unused_ignores = True
```
Comment (cqc-alec) [inline mypy.ini:22]: Thanks, can you remove the line above where it's set to `False` as well? (Not sure which applies if set twice.)

Comment (cqc-melf) [reply mypy.ini]: done

File: .github/workflows/docs/conf.py [hunk 2]
```diff
@@ -116,7 +116,6 @@ def correct_signature(
     signature: str,
     return_annotation: str,
 ) -> (str, str):
-
```
[the remark this answers is not in the document]
Comment (cqc-alec) [reply .github/workflows/docs/conf.py:119]: I think black has just had an update.

**Review by cqc-alec — APPROVED** [review]

**Pull request merged**